### PR TITLE
IO1-3978: sorted summary validator list

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/model/validator.model.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { IOperator } from '~app/common/stores/applications/SsvWeb';
```
Comment (IlyaVi) [inline src/app/model/validator.model.ts:1]: extract IOperator from store to operator.model.ts

File: src/app/components/applications/SSV/MyAccount/components/Validator/BulkActions/BulkComponent.tsx [hunk 2]
```diff
@@ -56,11 +57,11 @@ const BulkComponent = () => {
     setSelectedValidators(nextState);
   };
 
-  const onCheckboxClickHandler = (isChecked: boolean, publicKey: string) => {
+  const onCheckboxClickHandler = (isChecked: boolean, publicKey: string, validators: IValidator[]) => {
     if (isChecked && !selectedValidators.includes(publicKey)) {
-      setSelectedValidators((prevState: string[]) => [...prevState, publicKey]);
+      setSelectedValidators((prevState: string[]) => validators.map((validator: IValidator) => validator.public_key).filter((validatorPublicKey: string) => [...prevState, publicKey].includes(validatorPublicKey)));
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/Validator/BulkActions/BulkComponent.tsx:62]: simplify

**Review by IlyaVi — APPROVED** [review]

File: src/app/components/applications/SSV/MyAccount/components/Validator/BulkActions/BulkComponent.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/BulkActions/BulkComponent.tsx]: done

File: src/app/model/validator.model.ts (hunk 1 above)
Comment (axelrod-blox) [reply src/app/model/validator.model.ts:1]: done

**Pull request merged**